### DISCUSSION TITLE
Web deployment trouble. In need of guidance.

### DISCUSSION THREAD

**Discussion by 2-GO** [community/community#59367] [Other Feature Feedback, Questions, & Ideas]
Deployment Issue
Question

### Body

Hello Community, I am trying to deploy a portfolio website, but encountered some issues relating to the deployment. I believe the issue lies with the .html page, The published website shows a blank page, but I'm unsure how to proceed/fix. [[](https://github.com/2-GO/Portfolio) ](https://github.com/2-GO/Portfolio)

If anyone can provide some guidance/tips on how I can get it to work, I would really appreciate it. Thank you.

**Comment by radar-blip** [comment]
You could try putting your index.html in the main section (root) of your repository.
If that works, you may have to adjust any links accordingly.

**Reply by 2-GO** [reply]
Hi, @linuxlawson I believe that solution did not work for me, thank you
